docs/core/resilience/http-resilience.md

```diff
@@ -81,13 +81,13 @@ The preceding code adds the standard resilience handler to the <xref:System.Net.
 
 The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
 
-| Order | Strategy | Description |
-|--:|--|--|
-| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. |
-| **2** | Total request timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. |
-| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. |
-| **4** | Circuit breaker | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. |
-| **5** | Attempt timeout | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. |
+| Order | Strategy | Description | Defaults |
+|--:|--|--|--|
+| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. | Queue: 1,000<br>Permit: 0 |
+| **2** | Total timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. | Total timeout: 30s |
+| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. | Max retries: 3<br>Backoff: Constant<br>Use jitter: `false`<br>Delay:2s |
```

Comment:
```suggestion
| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. | Max retries: 3<br>Backoff: Exponential<br>Use jitter: `true`<br>Delay:2s |
```

`HttpRettryStrategyOptions` changes some defaults:

https://github.com/dotnet/extensions/blob/afde87b8812ef0b1dcee5db35cb3a49ca6dc9197/src/Libraries/Microsoft.Extensions.Http.Resilience/Polly/HttpRetryStrategyOptions.cs#L27